tests/e2e/tests/critical-path/database/import-databases.e2e.ts

```diff
@@ -126,7 +126,7 @@ test('Connection import modal window', async t => {
     await t.click(myRedisDatabasePage.removeImportedFileBtn);
     await t.expect(myRedisDatabasePage.importDbDialog.textContent).contains(defaultText, 'File not removed from import input');
 });
-test('Connection import from JSON', async t => {
+test.only('Connection import from JSON', async t => {
```

Comment:
.only

and it's better to add the test inside of 
"Exporting Standalone, OSS Cluster, and Sentinel connection types" not 'Connection import from JSON'
there both export and import tested
you can select decompressor for database, export it and then check that it was imported too